ui/intermittent-failures/DateRangePicker.jsx

```diff
@@ -116,31 +106,9 @@ class DateRangePicker extends React.Component {
 }
 
 DateRangePicker.propTypes = {
-  updateDates: PropTypes.func,
-  fetchData: PropTypes.func,
-  fetchFullBugData: PropTypes.func,
-  tree: PropTypes.string.isRequired,
-  bugId: PropTypes.oneOfType([
-    PropTypes.string,
-    PropTypes.number,
-  ]),
-  name: PropTypes.string.isRequired,
-  tableApi: PropTypes.string.isRequired,
-  graphApi: PropTypes.string.isRequired,
-  graphName: PropTypes.string.isRequired,
+  updateState: PropTypes.func.isRequired,
 };
 
 DateRangePicker.defaultProps = {
-  fetchData: null,
-  updateDates: null,
-  fetchFullBugData: null,
-  bugId: null,
+  updateState: null,
```

Comment:
ESLint on the Neutrino 9 branch reports:

```
error: propType "updateState" is required and should not have a defaultProp declaration
(react/require-default-props) at ui\intermittent-failures\DateRangePicker.jsx:109:3:
  107 |
  108 | DateRangePicker.propTypes = {
> 109 |   updateState: PropTypes.func.isRequired,
      |   ^
  110 | };
  111 |
  112 | DateRangePicker.defaultProps = {
```

Would you mind fixing in a followup?

Reply:
Will do!